src/main/scala/interplay/PlayBuildBase.scala

```diff
@@ -280,7 +280,30 @@ object PlayWhitesourcePlugin extends AutoPlugin {
   override def trigger: PluginTrigger = allRequirements
 
   override lazy val projectSettings = Seq(
-    whitesourceProduct := "Lightbend Reactive Platform"
+    whitesourceProduct := "Lightbend Reactive Platform",
+    whitesourceAggregateProjectName := {
```

Comment:
This doesn't include the module name, just the branch signifier.

Compare with Akka's implementation: https://github.com/akka/akka/blob/master/project/Whitesource.scala#L16

Reply:
Yep. I've forgotten this "detail". :-)